Doppler.ReportingApi/Models/DopplerTips.cs

```diff
@@ -0,0 +1,19 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+namespace Doppler.ReportingApi.Models
+{
+    public class DopplerTips
+    {
+        [JsonProperty("completed")]
+        public bool Completed { get; set; }
+
+        [JsonProperty("firstSteps")]
+        public List<Step> FirstSteps { get; set; }
+
+        [JsonProperty("notifications")]
+        public List<Notification> Notifications { get; set; }
```

Comment:
Es totalmente válido hacerlo con JsonProperty, incluso muchos dirán que "es la forma correcta" de hacerlo.

Así que acá no te pido un cambio, sino que te propongo una alternativa que estamos usando en otras APIs, y que consiste simplemente en utilizar los nombres de las propiedades en el formato que van a estar en el JSON:

```suggestion
        public bool completed { get; set; }

        public List<Step> firstSteps { get; set; }

        public List<Notification> notifications { get; set; }
```

Reply:
Me lastima ver propiedades en minúscula, pero como en este caso solo estoy retornando datos, no tiene sentido  tener las anotaciones.